content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -252,3 +252,92 @@ Currently, the KubeVirt CSI driver does not support volumes with block mode ther
+* Custom Local Disks
+* Secondary Disks
+
+{{% notice warning %}}
+The official upgrade procedure will not break clusters that already exist, however, **scaling cluster nodes will not lead to expected results**.  
+We require to update Machine Deployment objects and rotate machines right after the upgrade.
+{{% /notice %}}
+
+### Required migration steps
+
+Here we are going to cover manual steps that are required for smooth transition from technology preview to GA.
+
+#### Upgrade of KubeVirt Infrastructure cluster
+
+{{% notice warning %}}
+Updating of Kubernetes, KubeVirt and Containerized Data Imported should be done from N-1 to N release.
+{{% /notice %}}
+
+The k8s cluster and KubeVirt components must be in [scope of our supported versions](#requirements).  
+You can update k8s version by following [the official guide](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).  
+Or if you provisioned the cluster over KubeOne please follow [the update procedure](/kubeone/v1.5/tutorials/upgrading-clusters/).
+
+Next you can update KubeVirt control plane and Containerized Data Importer by executing:
+
+```shell
+export RELEASE=<SUPPORTED_RELEASE>
+kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-operator.yaml
+```
+
+```shell
+export RELEASE=<SUPPORTED_RELEASE>
+kubectl apply -f https://github.com/kubevirt/containerized-data-importer/releases/download/${RELEASE}/cdi-operator.yaml
+```
+
+{{% notice note %}}
+Refer to [the KubeVirt control update documentation](https://kubevirt.io/user-guide/operations/updating_and_deletion/#updating-kubevirt-control-plane)
+for further details about the update.
+{{% /notice %}}
+
+{{% notice warning %}}
+It is highly recommended to first test the upgrade on a staging environment.  
+It can happen that hosts where KubeVirt runs might need reconfiguration or packages update.
+{{% /notice %}}
+
+#### Update Machine Deployment
+
+{{% notice warning %}}
+If user clusters have working Load Balancers those can be unreachable after Machine Deployment rotation due to LB selector key change.  
+After this step, it is required to follow the [Update LoadBalancer selectors](#update-loadbalancer-selectors) guide to fix it.
+{{% /notice %}}
+
+Right after the upgrade it is required to update Machine Deployment object (this will trigger Machines rotation).  
+You can do it from the KKP Dashboard which is recommended approach as you will be guided with the possible options.
+
+![Machine Deployment Edit](/img/kubermatic/main/architecture/supported-providers/kubevirt/mc-edit.png?classes=shadow,border "Machine Deployment Edit")
+
+The alternative is to directly change Machine Deployment objects over `kubectl apply`.  
+Take a look into [the example](https://github.com/kubermatic/machine-controller/blob/v1.55.0/examples/kubevirt-machinedeployment.yaml) to see what has been changed.
```

Comment:
Can we link to machine-controller v1.56.0 here, since that is the release ending up in KKP 2.22?